### PR TITLE
WIP: Start adding data to person pages

### PR DESCRIPTION
**Pull request by tfgg**
image

### CODE REVIEW

**Comment by tfgg** [comment]
Wikipedia bios

image

**Comment by tfgg** [comment]
Currently the links on the right don't insert themselves at the start of the page on mobile. I need to learn some sort of media query voodoo to make that happen.

**Comment by tfgg** [comment]
Put buttons above in mobile view:
image

File: _layouts/person.html [hunk 1]
```diff
@@ -1,81 +1,156 @@
...
+    <p class='person__cv__text'>
+        {{ page.person.name}} has sent us {{ possessive|downcase }} CV. 
+
+        <a href="http://cv.democracyclub.org.uk/show_cv/{{ page.person.id }}">
+        Read {{ page.person.name}}'s CV now.
+        </a>
+    </p>
+    <div style='clear:both;'></div>
+  {% else %}
+    <p class='person__cv__text'>{{ page.person.name }} hasn't sent <a href='cv.democracyclub.org.uk'>Democracy Club CVs</a> {{ possessive|downcase }} CV.</p>
```
Comment (symroe) [inline _layouts/person.html]: `XX hasn't sent Democracy Club CVs her CV.` is a bit clunky – maybe `XXX hasn't sent us her CV`?  I don't think anyone will know why the extra words are in there.

Comment (symroe) [inline _layouts/person.html]: This is in keeping with the below: "We don't know of any leaflets" wording

**Pull request merged**